src/eventgenerator/aggregator/metricPoller_test.go

```diff
@@ -3,8 +3,8 @@ package aggregator_test
 import (
 	. "eventgenerator/aggregator"
 	. "eventgenerator/appmetric"
-	. "metricscollector/metrics"
-	. "metricscollector/server"
+	// . "metricscollector/server"
```

Comment:
I forgot to remove the commented line after testing. Will remove it

The ErrorResponse struct was moved to "models" package, eventgen  does not need to import from metricscollector any more